doc/api/crypto.md

```diff
@@ -1196,7 +1196,8 @@ rapidly.
 In line with OpenSSL's recommendation to use pbkdf2 instead of
 [`EVP_BytesToKey`][] it is recommended that developers derive a key and IV on
 their own using [`crypto.pbkdf2()`][] and to use [`crypto.createCipheriv()`][]
-to create the `Cipher` object.
+to create the `Cipher` object. A warning is emitted when counter mode (e.g. CTR,
+GCM or CCM) is used in `crypto.createCipher()` in order to avoid IV reuse.
```

Comment:
Maybe make this warning a little stronger, e.g. "a predictable IV undermines the security of counter-based ciphers."

Would it be worthwhile to mention that this API predates the addition of counter-based ciphers to OpenSSL, hence why it's not an error outright?

(Really, the more I think about it, the more I feel throwing an exception is the right thing to do.  If it weren't for backwards compatibility...)